apps/masterbots.ai/lib/email.ts

```
@@ -0,0 +1,106 @@
+import nodemailer from 'nodemailer'
+
+// * Get the app URL based on the environment for the password reset link
+function getAppUrl(): string {
+  switch (process.env.NEXT_PUBLIC_APP_ENV) {
+    case 'prod':
+      return 'https://masterbots.ai'
+    case 'test':
+      return 'https://dev.masterbots.ai'
+    case 'local':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)